packages/amplify_datastore/ios/Classes/FlutterDataStoreErrorMessage.swift

```diff
@@ -20,4 +20,5 @@ enum FlutterDataStoreErrorMessage: String {
     case INVALID_STATE = "AMPLIFY_INVALID_STATE"
     case MALFORMED = "AMPLIFY_REQUEST_MALFORMED"
     case QUERY_FAILED = "AMPLIFY_DATASTORE_QUERY_FAILED"
+    case DELETE_FAILED = "AMPLIFY_DATASTORE_DELETE_FAILED"
```

Comment:
enum cases are normally named in lowerCamelCase in swift. 

```suggestion
    case deleteFailed = "AMPLIFY_DATASTORE_DELETE_FAILED"
```

Reply:
If this enum is just used to namespace the error message I think you can do the following as mentioned in [here](https://google.github.io/swift/#apples-api-style-guidelines) 

> Declaring an enum without cases is the canonical way to define a “namespace” to group a set of related declarations, such as constants or helper functions. 

```swift
enum FlutterDataStoreErrorMessage {
    static let casting: String = "ERROR_CASTING_INPUT_IN_PLATFORM_CODE"
    static let sendingError: String = "AMPLIFY_REQUEST_FAILED_TO_SEND"
    ...
}
```

Reply:
This is a good suggestion but would require a lot of changes across the library.